### PR TITLE
Data section

### PR DESCRIPTION
**Pull request by justinsalamon**

### CODE REVIEW

File: book/data/introduction.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+"multi-track recording", since each instrument is recorded on a separate track of a digital audio workstation (DAW).
+2. The ability to programatically create mixtures from these stems for training and evaluation.
+
+
+## Data is a key component
+
+Your model can only be as good as the data used to train it (the [garbage in garbage 
+out](https://en.wikipedia.org/wiki/Garbage_in,_garbage_out) principle), and failing to use data (music) that is 
+representative of the data you plan to apply your model to ([sampling bias](https://en.wikipedia.org/wiki/Sampling_bias))
+It may be tempting to generate simple synthetic
```
Comment (ethman) [inline book/data/introduction.md:57]: Errr.... This is what Slakh is... 😂😂😂

File: book/data/introduction.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+representative of the data you plan to apply your model to ([sampling bias](https://en.wikipedia.org/wiki/Sampling_bias))
+It may be tempting to generate simple synthetic 
+stems or make some multi-track recordings in the lab and use these to train a model. However, there is little to no chance 
+that a model trained on such data will generalize to real-world music. Put simply, you need to train your model with data 
+that is representative of the type of data you plan to apply your model to once it is trained.
+
+## Data for source separation hard to obtain
+
+Due to copyright, it is hard to obtain and share music recordings for machine learning purposes. It is even harder to obtain
+multi-track recordings that include the isolated stems, as these are rarely made available by artists. Fortuntaely, the research
```
Comment (ethman) [inline book/data/introduction.md:65]: Typo: Fortuntaely --> Fortunately

File: book/data/introduction.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Since the master is not a linear sum of the stems, for training a source separation model we instead create our own mixture 
+by summing the stems directly to create a linear mixture. When training a source separation model, we provide this mixture 
+as input, the model outputs the estimated stems, and we compare these to the original stems that were used to create the 
+mixture. The difference between the estimated stems and the original stems is used to update the model parameters during 
+training:
+
+PLACEHOLDER: block diagram of training
+
+The difference between the estimated stems and original stems is also used to *evaluate* a trained source separation model,
```
Comment (ethman) [inline book/data/introduction.md:36]: We will have covered this briefly in my section. Probably no need to update this now, but something to keep in mind when we're doing our final pass.

**Review by pseeth — APPROVED** [review]
LGTM.

**Pull request merged**